MMM-Globe.js

```diff
@@ -36,6 +39,14 @@ Module.register("MMM-Globe",{
 		this.globe = null;
 		this.translatedSize = this.sizeTranslate(this.config.size);
         this.locations = this.config.locations;
+        this.viewAngle = this.convertDegreesToRadians(this.config.viewAngle);
+        if (this.config.dayLength >= 0) {
```

Comment:
Do we want to allow a daylength of 0?

Reply:
We probably do not. But it would quickly apparent that something was very wrong if they did. I'll look at it again and see if I want to add some zero checks.

Reply:
Sure thing. Give me a heads up here and I'll keep checking back.

Reply:
Will do. I've been kind of busy with another project lately. I'll try to get to it this weekend.

Reply:
I put a little logic to avoid zero rotation length. I'm leaving the two un-implemented variables for now. I hope to get to them in the near future. Right now, they don't hurt anything.